source/User_Guide/Marketing_Campaigns/senders.md

```diff
@@ -40,7 +39,7 @@ Adding a Sender
 
 
 {% info %}
-Your verification link is only valid for 48 hours. After that time you will need to restart the verification process.
+If you have a [verified whitelabel domain]({{root_url}}/User_Guide/Settings/Whitelabel/index.html) and your sender email address matches that domain exactly, your sender identity will automatically verify.
 {% endinfo %}
 
```

Comment:
The "Using Your Preset Sender Information In Your Campaigns" section is missing a lot of context too. Where would you add this information, and why?